circe/decoupled_pricing/home.Dockerfile

```diff
@@ -13,9 +14,9 @@ RUN mkdir -p DAG
 
 COPY circe/decoupled_pricing/start_home_controller.sh /start.sh
 
-ADD app_specific_files/dummy_app/configuration.txt DAG/DAG_application.txt
-ADD app_specific_files/dummy_app/input_node.txt DAG
-ADD app_specific_files/dummy_app/sample_input /
+ADD app_specific_files/dummy_app_multicast/configuration.txt DAG/DAG_application.txt
```

Comment:
These variables keep changing. We need a way to fix this annoying issue. What we can do is create Makefiles that generate the Dockerfiles based off the app-specific directory we're interested in.

Reply:
It is generated by all of the dockerfiles generator python scripts. That's why I recommended to put those .Dockerfile in ignore file. We actually don't need to commit it.

Reply:
Moreover, in the app_specific_folder, we currently have an empty folder.

Reply:
Instead, we need to then put all generated files into a build/ folder. That way we can exclude the build/ folders throughout the repository into the .gitignore file.